index.js

```diff
@@ -50,6 +50,24 @@ const response = function(fn, options = {}) {
     let res = null;
     const start = new Date().getTime();
     let statusCode = null;
+    // make sure query and path params always exist:
+    req.queryStringParameters = req.queryStringParameters || {};
+    req.pathParameters = req.pathParameters || {};
+    // preserve the original headers
+    req.headersRaw = Object.assign({}, req.headers);
+    // and normalize the headers so you don't have to worry about case:
+    req.headers = Object.keys(req.headersRaw).reduce((memo, key) => {
```

Comment:
any concern lowering them all?  the example had some that it skips

Reply:
They are preserving some of the caps ("WebSocket") etc.  We just want everything to be lowercase characters.

Reply:
any idea why they are doing that?

Reply:
ping

Reply:
I think they just like that style?  There's not really a compelling reason to prefer 'ALPN' over 'alpn' in arc-rapptor, it's just going into a user-provided function anyway.